### ISSUE TITLE
Does not work for batch_size > 1

### ISSUE THREAD

**Issue by flow2k** [twairball/keras_lstm_vae#9]
For batch_size = 2, I see `ValueError: could not broadcast input array from shape (2,3,13) into shape (1,3,13)`.

**Comment by JonasMedu** [comment]
yeah same here, Error is thrown, when the gradient is computed (@ tensor: decoder_mean).
I think that the architecture is flawed, not sure how to fix it yet though
